.gitignore

```diff
@@ -7,6 +7,9 @@
 # Ignore bundler config.
 /.bundle
 
+# Ignore deface precompiled views
+app/compiled_views
```

Comment:
We propose to avoid adding compiled_views in git. This way, you can run your application in development without both "deface rendering" and "deface precompiled files". 

We add the precompilation in entrypoint to be sure deface is precompiled before starting. 
This precompilation MUST be done on each redeploy ;)